src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -344,6 +347,14 @@ private synchronized void regenerateAITasks() {
 				L10N.t("elementgui.living_entity.condition_solid_bounding_box"), AbstractProcedureSelector.Side.BOTH,
 				L10N.checkbox("elementgui.common.enable"), 160,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity"));
+		visualScale = new NumberProcedureSelector(this.withEntry("entity/visual_scale"), mcreator,
+				L10N.t("elementgui.living_entity.visual_scale"), AbstractProcedureSelector.Side.CLIENT,
+				new JSpinner(new SpinnerNumberModel(1, 0.01, 1024, 0.05)), 308, Dependency.fromString(

```

Comment:
Minimum is not a multiple of step size, same for the other one.

Reply:
Is it fine to keep this one unchanged?
```suggestion
				new JSpinner(new SpinnerNumberModel(1, 0.01, 1024, 0.01)), 308, Dependency.fromString(
```